Detections/SecurityEvent/UserAccountEnabledDisabled_10m.yaml

```diff
@@ -85,19 +85,26 @@ query: |
   | where tolong(TimeDelta) >= threshold
   | project TimeDelta, EnableTime, EnableEventID, EnableActivity, Computer, TargetAccount, TargetSid, UserPrincipalName, AccountUsedToEnable, SIDofAccountUsedToEnable, 
   DisableTime, DisableEventID, DisableActivity, AccountUsedToDisable, SIDofAccountUsedToDisable
-  | extend timestamp = EnableTime, AccountCustomEntity = AccountUsedToEnable, HostCustomEntity = Computer
+  | extend timestamp = EnableTime
+  | extend HostName = tostring(split(Computer, ".")[0]), DomainIndex = toint(indexof(Computer, '.'))
+  | extend HostNameDomain = iff(DomainIndex != -1, substring(Computer, DomainIndex + 1), Computer)
+  | extend AccountName = tostring(split(AccountUsedToEnable, "@")[0]), AccountUPNSuffix = tostring(split(AccountUsedToEnable, "@")[1])
 entityMappings:
   - entityType: Account
     fieldMappings:
-      - identifier: FullName
-        columnName: AccountCustomEntity
+      - identifier: Name
+        columnName: AccountName
```

Comment:
AccountUsedToEnable and AccountUsedToDisable are actually built from separated entity types already. This is a legacy method.  You can get what you need by the following:
| extend TargetUserName = tostring(EventData.TargetUserName), TargetDomainName = tostring(EventData.TargetDomainName)

Then entity map these to Name and NTDomain.

Also, please map another entity to Account for AccountUsedToDisable which will be SubjectUserName and SubjectDomainName.